### DISCUSSION TITLE
Can Kong wrap cloud run?

### DISCUSSION THREAD

**Discussion by terekete** [Kong/kong#12111] [Help]
Looking to use cloud run for it ease and flexibility, is there a way for Kong to wrap Cloudrun?

**Comment by chobits** [comment]
Could you give more details about your environment instaling kong in cloudrun?

**Comment by terekete** [comment]
Cloud run would be a single instance Cloud Run endpoint, not hosted on GKE. Wondering what would be required to use Kong API gateway with such an instance. I can't seem to find any examples of this.

**Reply by chobits** [reply]
yeah, there is no example in our official documentation, there are too many environments in real world, so not all of them can be covered.
You can start by deploying the Kong gateway firstly. If you encounter any problems, feel free to post them here or file a issue for assistance.
